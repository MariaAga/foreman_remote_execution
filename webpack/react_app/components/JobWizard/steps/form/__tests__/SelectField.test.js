import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
import { SelectField } from '../SelectField';

const fixtures = {
  'renders with props': {
    label: 'grouped select',
    fieldId: 'field-id',
    options: ['Commands'],
    value: 'Commands',
    setValue: jest.fn(),
  },
};

describe('SelectField', () => {
  describe('rendering', () =>
    testComponentSnapshotsWithFixtures(SelectField, fixtures));
});
