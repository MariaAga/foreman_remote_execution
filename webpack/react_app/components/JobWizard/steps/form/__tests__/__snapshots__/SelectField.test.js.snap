// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectField rendering renders with props 1`] = `
<FormGroup
  fieldId="field-id"
  label="grouped select"
>
  <Select
    aria-label="Your title"
    aria-labelledby=""
    className="without_select2"
    clearSelectionsAriaLabel="Clear all"
    createText="Create"
    customBadgeText={null}
    customContent={null}
    direction="down"
    favorites={Array []}
    favoritesLabel="Favorites"
    hasInlineFilter={false}
    id="horzontal-form-title"
    inlineFilterPlaceholderText={null}
    inputIdPrefix=""
    isCreatable={false}
    isDisabled={false}
    isGrouped={false}
    isOpen={false}
    isPlain={false}
    maxHeight="45vh"
    menuAppendTo="inline"
    name="horizontal-form-title"
    noResultsFoundText="No results found"
    onClear={[Function]}
    onCreateOption={[Function]}
    onFilter={null}
    onSelect={[Function]}
    onToggle={[Function]}
    ouiaSafe={true}
    placeholderText=""
    removeSelectionAriaLabel="Remove"
    selections="Commands"
    toggleAriaLabel="Options menu"
    toggleIcon={null}
    toggleId={null}
    typeAheadAriaLabel=""
    variant="single"
    width=""
  >
    <SelectOption
      className=""
      component="button"
      index={0}
      inputId=""
      isChecked={false}
      isDisabled={false}
      isFavorite={null}
      isNoResultsOption={false}
      isPlaceholder={false}
      isSelected={false}
      key="0"
      keyHandler={[Function]}
      onClick={[Function]}
      sendRef={[Function]}
      value="Commands"
    />
  </Select>
</FormGroup>
`;
